src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -694,14 +679,15 @@ Transactor::operator()()
         }
 
         ctx_.apply(terResult);
-        // VFALCO NOTE since we called apply(), it is not
-        //             okay to look at view() past this point.
+        // since we called apply(), it is not okay to look
+        // at view() past this point.
     }
 
-    auto const result =
-        std::make_pair(terResult, didApply);
-    log(result, j_);
-    return result;
+    JLOG(j_.trace) <<
+        "apply: " << transToken(terResult) <<
+        ", " << (didApply ? "true" : "false");
```

Comment:
Could use `std::boolalpha`, though that does change that format flag for future calls. Could change it back with `std::noboolalpha`.